packages/@guardian/source-foundations/src/index.ts

```diff
@@ -15,21 +17,10 @@ export { breakpoints } from './breakpoints/breakpoints';
 export { between, from, until } from './mq/mq';
 
 // palette
+export { palette };
+
+// deprecated exports - to be removed in v5
```

Comment:
Just in case this doesn't happen in the very next major version bump 😅
```suggestion
// deprecated exports - to be removed in future major version
```